pkgs/development/python-modules/joblib/default.nix

```diff
@@ -10,21 +11,31 @@
 
 buildPythonPackage rec {
   pname = "joblib";
-  version = "0.12.4";
+  version = "0.13.2";
 
   # get full repository inorder to run tests
   src = fetchFromGitHub {
     owner = "joblib";
     repo = pname;
     rev = version;
-    sha256 = "06zszgp7wpa4jr554wkk6kkigp4k9n5ad5h08i6w9qih963rlimb";
+    sha256 = "11bspmm2is8akcld2kh6bgp0in1ggdy91qsk5zqybzgapc8b4zdf";
   };
 
+  patches = [
+    # python-lz4 compatibility
+    (fetchpatch {
+      url = "https://github.com/joblib/joblib/pull/847.patch";
```

Comment:
Fetching PR patches is dangerous because the content can change.

Reply:
How about simply deselecting the test?

Reply:
I know, see my comment above.